### ISSUE TITLE
[nssdca-delivery] urn:nasa:pds:juno_jiram

### ISSUE THREAD

**Issue by tinagueth** [NASA-PDS/operations#451]
### Discipline Node Information

* **Delivering Node**: PDS_ATM

* **NSSDCA Delivery Package**: http://atmos.nmsu.edu/~tgueth/juno_jiram_v1.1_20231102.tar.gz

* **Validation report**: http://atmos.nmsu.edu/~tgueth/juno_jiramv14.txt

NOTE: If you have multiple delivery packages, we strongly encourage you to submit these in batches of 3 to 10 per issue with one ZIP file of the packages and another ZIP file of the validation reports. Please use a descriptive title, such as "Node Mission misc batch #".

----

### Engineering Node Process

See the internal EN process at https://pds-engineering.jpl.nasa.gov/content/nssdca_interface_process

**Comment by oddes** [comment]
@tinagueth following needs to be corrected and resubmitted. 
1. Version needs to 3.3.
2. Data Content Validation       off

**Comment by lylehuber** [comment]
3.3 was apparently released on Sept 25. It would have been nice to be notified. But also requiring/demanding it on such a quick turnaround? Let's just say "not cool".

**Comment by smclaughlin7** [comment]
@tinagueth This delivery contains two SIPs for the v1.1 of the JUNO JIRAM bundle.  I compared the manifest files for both SIPs and determined there are only 6 differences between them:

**Product:**  urn:nasa:pds:juno_jiram:calibration:jir_img_l_responsivity::3.0
- File changed:
    https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_L_RESPONSIVITY.xml
- File added:
    https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_L_RESPONSIVITY_V03.LBL

**Product:**  urn:nasa:pds:juno_jiram:calibration:jir_img_m_responsivity::3.0
- File changed:
    https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_M_RESPONSIVITY.xml
- File added:
    https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_M_RESPONSIVITY_V03.LBL

**Product:**  urn:nasa:pds:juno_jiram:calibration:jir_img_tot_responsivity::3.0
- File changed:
    https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_TOT_RESPONSIVITY.xml
- File added:
    https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_TOT_RESPONSIVITY_V03.LBL

The newest SIP, urn:nasa:pds:system_bundle:product_sip_deep_archive:juno_jiram_v1.1_20231101, appears to have been generated on the latest JIRAM bundle v1.1. Since both SIPs are nearly identical, would it be acceptable if EN @oddes only post the newest SIP for the NSSDCA to process?  Thanks!

**Comment by lylehuber** [comment]
Yes that’s fine.

Lyle


On Nov 10, 2023, at 6:13 AM, smclaughlin7 ***@***.***> wrote:

WARNING This email originated external to the NMSU email system. Do not click on links or open attachments unless you are sure the content is safe.


@tinagueth<https://github.com/tinagueth> This delivery contains two SIPs for the v1.1 of the JUNO JIRAM bundle. I compared the manifest files for both SIPs and determined there are only 6 differences between them:

Product: urn:nasa:pds:juno_jiram:calibration:jir_img_l_responsivity::3.0

  *   File changed:
https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_L_RESPONSIVITY.xml
  *   File added:
https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_L_RESPONSIVITY_V03.LBL

Product: urn:nasa:pds:juno_jiram:calibration:jir_img_m_responsivity::3.0

  *   File changed:
https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_M_RESPONSIVITY.xml
  *   File added:
https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_M_RESPONSIVITY_V03.LBL

Product: urn:nasa:pds:juno_jiram:calibration:jir_img_tot_responsivity::3.0

  *   File changed:
https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_TOT_RESPONSIVITY.xml
  *   File added:
https://pds-atmospheres.nmsu.edu/PDS/data/PDS4/juno_jiram_bundle/calibration/JIR_IMG_TOT_RESPONSIVITY_V03.LBL

The newest SIP, urn:nasa:pds:system_bundle:product_sip_deep_archive:juno_jiram_v1.1_20231101, appears to have been generated on the latest JIRAM bundle v1.1. Since both SIPs are nearly identical, would it be acceptable if EN @oddes<https://github.com/oddes> only post the newest SIP for the NSSDCA to process? Thanks!

**Comment by smclaughlin7** [comment]
@lylehuber Thanks, Lyle!

@oddes @c-suh Please post only the newest SIP in this delivery package, SIP LID:

- urn:nasa:pds:system_bundle:product_sip_deep_archive:juno_jiram_v1.1_20231101

**Comment by tinagueth** [comment]
@smclaughlin7 @oddes @c-suh I was doing a check-up on all the recent submissions to NSSDCA and when I got to this SIP LID, it states:
SIP LIDVID: urn:nasa:pds:system_bundle:product_sip_deep_archive:juno_jiram_v1.1_20231101:: All versions

Not Found: 

Remarks:  LID(VID) may be incorrect or initial submission was rejected.

will this change based on the above comments? I am trying to keep on top of all the submissions and any issues they have/had for our own documents. Thanks!

**Comment by c-suh** [comment]
@tinagueth and @lylehuber, apologies for any confusion. We have not processed this submission yet. My colleague, @oddes, mentioned 2 items, the first of which we highly encourage (upgrading to and using the [latest version](https://github.com/NASA-PDS/validate/releases/latest) of the Validate tool, currently v3.3.1). We recognize the effort this can take and appreciate your willingness to do so, trusting that the bug fixes and improvements in every release make it worthwhile.

The second item is regarding the Data Content Validation parameter, which must be enabled when validating the products. Please rerun the Validate tool on the newest SIP (i.e., urn:nasa:pds:system_bundle:product_sip_deep_archive:juno_jiram_v1.1_20231101) and do **not** use the options `-D` or  `--skip-content-validation`. Once this is done and the validation report is resubmitted (in this same ticket please), we will process and post this for NSSDCA. 

Thank you!

**Comment by jordanpadams** [comment]
@tinagueth can you re-run validate with content validation enabled and upload the report so we can ensure the SIP is valid? (remove `--skip-content-validation` flag from execution)

**Comment by lylehuber** [comment]
Validate is currently running. It kept failing because of the out-of-memory issue. I have applied the fix. It should finish around Christmas Day.

**Comment by jordanpadams** [comment]
@lylehuber copy! Thanks! sorry for the hassle here.

**Comment by lylehuber** [comment]
No problem, thanks for resolving the NaN issue. I suspect that now that we can do content validation, we might encounter some other issues.

**Comment by c-suh** [comment]
@lylehuber a quick check on the re-run of validate with content validation enabled. Did it reveal or cause other issues as you anticipated?

**Comment by lylehuber** [comment]
Yes, it revealed other issues. We should have those cleaned up in the few weeks.

On Feb 2, 2024, at 2:23 PM, Catherine Suh ***@***.***> wrote:

WARNING This email originated external to the NMSU email system. Do not click on links or open attachments unless you are sure the content is safe.


@lylehuber<https://github.com/lylehuber> a quick check on the re-run of validate with content validation enabled. Did it reveal or cause other issues as you anticipated?

**Comment by c-suh** [comment]
Okay; thank you for the update and sorry for the trouble!

**Comment by tinagueth** [comment]
I talked to Lyle about this bundle and he agrees with me that at this stage, we should just "cancel" this Juno jiram submission. Notes of the issues that arose during this submission were taken and tried to be resolved with newer submissions. And we do have this bundle backed it up via AWS. Thanks

**Comment by smclaughlin7** [comment]
@tinagueth Thank you following up on this SIP package where EN found some issues and did not submit to NSSDCA:

- urn:nasa:pds:system_bundle:product_sip_deep_archive:juno_jiram_v1.1_20231101

@c-suh Since @tinagueth and Lyle agreed this SIP package has been superseded and should not be deep-archived, this ticket can be closed. Thanks!
